### ISSUE TITLE
Prep for 1.0.0

### ISSUE THREAD

**Issue by atesgoral** [atesgoral/hrm-level-data#20]
@nrkn
- Agree we're happy with the latest schema
- Proof-read instructions
- Checking level data against actual game once more is tedious and may not be necessary
- Adding more tests can be done post 1.0.0 as a patch release

**Comment by nrkn** [comment]
Schema looks good - I like that you've made omission of the boolean properties mean that they're false, it's easy to test programatically and makes for a more concise json

Do we want to enforce that inbox should contain at least one thing? The program will immediately exit without doing anything otherwise. Though that kinda is OK maybe?

**Comment by nrkn** [comment]
Have you started proof-reading instructions? I am happy to do this now but don't want to duplicate effort so will wait on your say-so

**Comment by atesgoral** [comment]
No, I haven't started proof-reading. I thought it would be nice to have a different set of eyes have a look at it. :)

**Comment by nrkn** [comment]
:+1: I'll do that now

**Comment by nrkn** [comment]
I only found one thing!

**Comment by atesgoral** [comment]
Thanks!

**Comment by atesgoral** [comment]
One question on the `floor` property -- should it be optional so that we don't have to use `{}` for levels without floor setups? Also, some floors don't have any default tiles. Should `tiles` also be optional?

**Comment by nrkn** [comment]
Yes and yes - hrm-cpu correctly handles both cases

(actually I suspect in the latter case it assigns to memory addresses called 'columns' and 'rows' but it doesn't cause any tests to fail - I'll make a note to fix this later)

**Comment by atesgoral** [comment]
OK I'll adjust the schema and trim the dummy `floor` and `tiles` properties. #23
